### ISSUE TITLE
Separate schedule days into a list

### ISSUE THREAD

**Issue by justingallagher** [ScottyLabs/course-api#37]
In the list of times for a lecture/section, days of the week are listed like this:

`"days" : "UT"`

It should be like this:

`"days" : ["U", "T"]`

**Comment by justinchuby** [comment]
Also can we split the room as well? i.e. `"WEH 8427"` becomes `{"building": "WEH", "room": "8427}`

**Comment by justinchuby** [comment]
What about also having integer representation for days? I feel that's more standard? For example, we can have
```
{
"days_int" : [0, 2, 4],
"days" : "MWF"
}
# or [1,3,5] according to ISO-8601? I'm not sure which is more commonly used, but we can specify one.
```
@justingallagher Do you think we can keep `"days"` as string and add the int field? I am not sure if that's better.

**Comment by justingallagher** [comment]
@justinchuby Can you think of a use case for splitting the rooms from the building? I'm not sure when that would be really needed. In addition there are some outliers in the data, for example how should we record "IA CP AUD", "UC ACT RM" or "GYM MGYM" (which are all "rooms" in the data).

As for the days, we probably should use 0-6 for Sun - Sat as that's what we do for the dining API. I don't think we need to keep the days string, it can be reproduced relatively easily from the list.

**Comment by justinchuby** [comment]
Yes. For example I needed to filter base on location of the classes. I would search for a building, say, "DH", and retrieve a list of classes happening in that building. The data is also easier to use because we don't need to split the string manually if we need to display the information separately. (Although it would just be two lines of code, having the information separated at first still makes the data clearer, I think.) 

In the cases of "UC ACT RM" and "GYM MGYM", we can just split at the first space, i.e. "UC", "ACT RM" and "GYM", "MGYM". Or, treat the "GYM" one specially, keeping "GYM MGYM" in the building field and leave the room field blank.

I think 0-6 for Sun - Sat is good.

**Comment by justingallagher** [comment]
Alright, I'll probably split it into "building" and "room" by splitting on the first space.

**Comment by justingallagher** [comment]
Resolved with #39
